app/frontend/src/services/formService.js

```diff
@@ -93,14 +93,12 @@ export default {
 
   /**
   * @function getSubmission
-  * Get the form data
-  * @param {string} formId The form uuid
-  * @param {string} versionId The form version uuid
+  * Get the form data + version + submission data
   * @param {string} submission The form submission identifier
   * @returns {Promise} An axios response
   */
-  getSubmission(formId, versionId, submissionId) {
-    return appAxios().get(`${ApiRoutes.FORMS}/${formId}/versions/${versionId}/submissions/${submissionId}`);
+  getSubmission(submissionId) {
+    return appAxios().get(`/submissions/${submissionId}`);
```

Comment:
So won't be off "forms"?
`${ApiRoutes.FORMS}` intended to be removed?

Reply:
yes, no need for form or version

Reply:
This may need to be moved into its own service js file if its root is different. Can defer this for now.